TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/EditRoute/EditRouteCommonPageModel.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+namespace TeachingRecordSystem.SupportUi.Pages.RoutesToProfessionalStatus.EditRoute;
+
+public abstract class EditRouteCommonPageModel(TrsLinkGenerator linkGenerator, ReferenceDataCache referenceDataCache)
+    : PageModel
+{
+    protected TrsLinkGenerator LinkGenerator => linkGenerator;
+
+    protected ReferenceDataCache ReferenceDataCache => referenceDataCache;
+
+    public JourneyInstance<EditRouteState>? JourneyInstance { get; set; }
+
+    [FromQuery]
+    public bool FromCheckAnswers { get; set; }
+
+    [FromRoute]
+    public Guid QualificationId { get; set; }
+
+    public string? PersonName { get; set; }
+
+    public Guid PersonId { get; set; }
+
+    public RouteToProfessionalStatusType Route { get; set; } = null!;
```

Comment:
```suggestion
    public RouteToProfessionalStatusType RouteType { get; set; } = null!;
```

Reply:
The equivalent property is called `Route` on the add route page base type `AddRoutePostStatusPageModel` - if we're renaming it here, we should rename it there as well